app/src/main/java/com/tvtracker/MainActivity.kt

```diff
@@ -83,7 +83,7 @@ class MainActivity : ComponentActivity() {
                             LazyColumn(state = listState) {
                                 itemsIndexed(mediaItems) { index, item ->
                                     viewModel.onChangeScrollPosition(index)
-                                    if((index + 1) >= (viewModel.page * viewModel.PAGE_SIZE) && !viewModel.loading){
+                                    if((index + 1) >= (viewModel.page * viewModel.getPageSize()) && !viewModel.loading){
```

Comment:
I created a getPageSize() function in order to access the companion object inside of view model.